cmd.coffee

```diff
@@ -45,19 +60,24 @@ process_file = (file)->
       contract : argv.contract
       replace_enums_by_nats: not argv.disable_enums_to_nat
       prefer_erc721: argv.prefer_erc721
+      keep_dir_structure: argv.dir != null
   }
   new_ast = ast_transform.pre_ti new_ast, opt
   new_ast = type_inference new_ast, opt
   new_ast = ast_transform.post_ti new_ast, opt
+
   code = translate new_ast, opt
   code += """\n (* this code is generated from #{file} by sol2ligo transpiler *)"""
-  
   if argv.outfile
     name = outfile.name
     if outfile.ext
       name += outfile.ext
     else
       name += ".ligo"
+    name = path.join outfile.dir, name
+    if outfile.dir
+      execSync "mkdir -p #{outfile.dir}"
```

Comment:
missing shellescape

Reply:
I just took your bit of code. I don't know what consequences it may lead to.
https://github.com/madfish-solutions/sol2ligo/blob/a02fd9cd8ce80a2cf4588f412416d72278463bce/src/import_resolver.coffee#L31

Reply:
That case should be also fixed
https://github.com/madfish-solutions/sol2ligo/issues/251